### ISSUE TITLE
Bug: Validator doesn't recognize aspect-ratio property

### ISSUE THREAD

**Issue by tmb-github** [w3c/css-validator#291]
See [MDN article on aspect-ratio property](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio)

```
/*
It's unaware of the aspect-ratio property:
https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio
*/
@supports (aspect-ratio: 1 / 1) {
    .aspect-ratio-1-1 {
        aspect-ratio: 1 / 1
    }
}
@supports not (aspect-ratio: 1 / 1) {
    .aspect-ratio-1-1 {
        display: block;
    }
}
```

**Comment by MichaIng** [comment]
Duplicates #287. But good to have it given a bump 🙂.

**Comment by ylafon** [comment]
Fixed by #308
